Recipes Project.py

```diff
@@ -1,3 +1,18 @@
+#Recipe
+#marayah
+
+def recipes():
+    """
+    Description of the function
+    Input:
+    Ingredients that are available to user
+
+    Output:
+    All recipes that require only those ingredients
+    :return:
```

Comment:
I'm not sure what :return: does. Do you mean to explain what the package returns, here?